### PR TITLE
Fix elasticsearch mounts

### PR DESCRIPTION
**Pull request by shaidar**
This just fixes a few things that Mark had already been working on regarding the ES7 deployment.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -50,6 +50,7 @@ repos:
       args:
         - "--config=setup.cfg"
         - '--extend-ignore=D1'
+        - '--max-string-usages=10'
```
Comment (blarghmatey) [inline .pre-commit-config.yaml:53]: This should be removed. Any configuration settings for flake8 should be managed via setup.cfg

File: src/ol_infrastructure/infrastructure/elasticsearch/__main__.py [hunk 2]
```diff
@@ -125,16 +127,16 @@
     ingress=[
         ec2.SecurityGroupIngressArgs(
             protocol="tcp",
-            from_port=9200,
-            to_port=9200,
+            from_port=9200,  # noqa: WPS432
```
Comment (blarghmatey) [inline src/ol_infrastructure/infrastructure/elasticsearch/__main__.py:130]: For these kinds of magic numbers I actually created a module (I think called magic_numbers) under utils to hold those values and then import and use the descriptive names in these locations. That's preferable to sprinkling `noqa` statements everywhere we use a number, at least for well-known ports, etc.

**Review by blarghmatey — CHANGES_REQUESTED** [review]

File: src/ol_infrastructure/infrastructure/elasticsearch/__main__.py (hunk 2 above)
Comment (shaidar) [reply src/ol_infrastructure/infrastructure/elasticsearch/__main__.py:130]: That util is in the configuration repo, so leaving this as is for now.

**Review by blarghmatey — APPROVED** [review]

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**